test/integration/io_waitreadable_test.rb

```diff
@@ -50,7 +50,7 @@ def io_waitreadable_sleep
   def test_slow_io_waitreadable_sleep
     # This test hits IO::WaitReadable exception multiple times
     # For that, this test must run longer than 1 second
-    assert measure_io_waitreadable_sleep > 1
+    assert_operator measure_io_waitreadable_sleep, :>, 1
```

Comment:
Hi @iuri-gg,
thanks for posting that.
I have to check all the tests for styleguide puposes.